java/flight/src/main/java/org/apache/arrow/flight/FlightInfo.java

```diff
@@ -17,13 +17,22 @@
 
 package org.apache.arrow.flight;
 
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.nio.ByteBuffer;
+import java.nio.channels.Channels;
 import java.util.List;
 import java.util.stream.Collectors;
 
 import org.apache.arrow.flight.impl.Flight;
 import org.apache.arrow.flight.impl.Flight.FlightGetInfo;
+import org.apache.arrow.vector.ipc.ReadChannel;
+import org.apache.arrow.vector.ipc.WriteChannel;
+import org.apache.arrow.vector.ipc.message.MessageSerializer;
 import org.apache.arrow.vector.types.pojo.Schema;
 
+import com.fasterxml.jackson.databind.util.ByteBufferBackedInputStream;
```

Comment:
Yes, I didn't like having to use a random Jackson class; happy to provide some implementation of it. Really it would be nice to have it in the JDK though.